.eslintrc.json

```diff
@@ -71,12 +72,41 @@
         "pathGroupsExcludedImportTypes": ["react"]
       }
     ],
+    "max-depth": ["warn", 1],
```

Comment:
What do you think about increasing it a little bit to 2 or 3?

Reply:
Agree, let's set it to 2, shouldn't affect code complexity much